mintupdate.pot

```diff
@@ -1458,7 +1458,7 @@ msgid "Show a warning if system snapshots are not set up"
 msgstr ""
 
 #: usr/share/linuxmint/mintupdate/preferences.ui.h:24
-msgid "Don't suggest to switch to a local mirror"
+msgid "Don't suggest switching to a local mirror"
```

Comment:
You'll need to update [this line](https://github.com/linuxmint/mintupdate/blob/master/usr/share/linuxmint/mintupdate/preferences.ui#L865) in preferences.ui as well.

Reply:
Ok, I have updated the line in the file.